src/widgets/_scales.scss

```
@@ -131,13 +142,15 @@ scale {
         highlight.top,
         fill.top {
             border-radius: rem(12px) 0 0 rem(12px);
-            margin: 0 0 0 rem(-6px);
+            // Account for border width
+            margin: -1px 0 -1px calc(#{rem(-6px)} -1px);
```

Nitpick: if it's subtraction and not passing `-1px` in, there should be spaces around the `-`.

```suggestion
            margin: -1px 0 -1px calc(#{rem(-6px)} - 1px);
```